src/WebJobs.Extensions.DurableTask/LinuxConsumptionLogger.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    internal class LinuxConsumptionLogger : LinuxAppServiceILogger
```

Comment:
You may need some extra metadata for LinuxConsumption, as it does not handle things like Tenant and RoleInstance for you like windows and linux dedicated do.

If you make `GenerateJsonStringToLog` virtual (or some subset of the method), you can override it here to add this extra metadata.

Reply:
Right. I was looking at your EasyAuth PR and was not sure how you were populating those _extra_ fields. Do those correspond to some enviroment variables that I can parse?